### PR TITLE
Add docstrings to spider classes 2

### PR DESCRIPTION
**Pull request by aguilerapy**
#347

### CODE REVIEW

**Comment by jpmckinney** [comment]
@yolile @romifz Can this be reviewed this week?

File: kingfisher_scrapy/spiders/moldova_old.py [hunk 1]
```diff
@@ -3,6 +3,13 @@
 
 
 class MoldovaOld(SimpleSpider):
+    """
+    Bulk download documentation
+      http://opencontracting.date.gov.md/downloads
+    Spider arguments
+      sample
+        Download only data released on 2017.
```
Comment (romifz) [inline kingfisher_scrapy/spiders/moldova_old.py:11]:
```suggestion
        Downloads a single JSON file containing data for 2017
```

File: kingfisher_scrapy/spiders/scotland.py [hunk 2]
```diff
@@ -5,6 +5,13 @@
 
 
 class Scotland(SimpleSpider):
+    """
+    API documentation
+      https://api.publiccontractsscotland.gov.uk/v1
+    Spider arguments
+      sample
+        Download only 21 releases.
```
Comment (romifz) [inline kingfisher_scrapy/spiders/scotland.py:13]:
```suggestion
        Downloads packages for releases dated one year ago, for each notice type available
```

File: kingfisher_scrapy/spiders/uk_contracts_finder.py [hunk 3]
```diff
@@ -5,6 +5,11 @@
 
 
 class UKContractsFinder(SimpleSpider):
+    """
+    Spider arguments
+      sample
+        Download only 100 release packages.
```
Comment (romifz) [inline kingfisher_scrapy/spiders/uk_contracts_finder.py:11]:
```suggestion
        Downloads the first page of release packages returned by the main endpoint
```

File: kingfisher_scrapy/spiders/uk_fts.py [hunk 4]
```diff
@@ -5,6 +5,11 @@
 
 
 class UKContractsFinder(LinksSpider):
+    """
+    Spider arguments
+      sample
+        Download only 1 release package.
```
Comment (romifz) [inline kingfisher_scrapy/spiders/uk_fts.py:11]:
```suggestion
        Downloads the first release package returned by the main endpoint
```

**Pull request merged**